terraform/terraform_cloud.tf

```diff
@@ -0,0 +1,48 @@
+# Configure Terraform Cloud, with Terraform
+#
+# Terraform Cloud is used only for one thing: to store the terraform state.
+#
+locals {
+  # FIXME: add all the admins of the org
+  tfe_owners = {
+    zimbatm = "zimbatm@zimbatm.com"
+  }
+
+  tfe_org = tfe_organization.nix-communiyt.name
+}
+
+# Org setup
+resource "tfe_organization" "nix-community" {
+  name = "nix-community"
+  # FIXME: host our own email
```

Comment:
Do you mean running an email server ourselves?

Reply:
Hopefully not. A forwarding address service is enough.